src/web/browser/coreVitals/init.ts

```diff
@@ -0,0 +1,10 @@
+import '../webpackPublicPath';
+import { startup } from '@root/src/web/browser/startup';
+import { coreVitals } from './coreVitals';
+
+const init = (): Promise<void> => {
+	coreVitals();
+	return Promise.resolve();
+};
```

Comment:
You could use [async](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function) here

```typescipt
const init = async (): Promise<void> => {
  coreVitals();
  return;
};
```

Reply:
I guess this also where we would set sampling?

Reply:
> I guess this also where we would set sampling?

+1

Reply:
Yeah, perhaps I'll add that in and comment it out until it goes fully live

Reply:
@oliverlloyd I didn't use async as none of the other init files do, so I was keeping to the pattern. I can add it in if you think it would be better?

Reply:
I used it in the Sentry init file based on a PR review I got there. I don't think it matters a great deal if you use it or not but the review comment I got improved my knowledge of what `async` is and does!

Reply:
👍